src/views/EnemiesList/EnemiesList.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import Enemies from '../../components/Enemies/Enemies';
+import { fetchEnemies } from '../../services/Enemies';
+import './EnemiesList.css';
+
+export default function EnemiesView() {
+  const [enemies, setEnemies] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchEnemies();
+      setEnemies(data);
+      setLoading(false);
+    };
+    if (loading) {
```

Comment:
No need to set our fetch conditional on loading. This function should run once after the component has mounted